stdlib/Dates/test/runtests.jl

```diff
@@ -0,0 +1,17 @@
+module DateTests
+
+using Test
+using Dates
+
+include("accessors.jl")
+include("adjusters.jl")
+include("query.jl")
+include("periods.jl")
+include("ranges.jl")
+include("rounding.jl")
+include("types.jl")
+include("io.jl")
+include("arithmetic.jl")
+include("conversions.jl")
```

Comment:
Shouldn't we have a `@testset "Dates" begin` wrapping all of this?

Reply:
Sure.